src/client/components/GameCardComponent/FormComponent/FormComponent.style.css

```diff
@@ -0,0 +1,34 @@
+@import url('https://fonts.googleapis.com/css2?family=Rammetto+One&family=Roboto&display=swap');
+
+html {
+  background-color: #fba85c;
+  background-image: url('../../../assets/images/background.png');
+  background-blend-mode: multiply;
+  background-repeat: no-repeat;
+}
+
+.form-input {
+  position: absolute;
+  background: #ffffff;
+  position: relative;
```

Comment:
also, you have position relative overwriting it here anyway 😬

Remove the above position